x-pack/plugin/async-search/src/internalClusterTest/java/org/elasticsearch/xpack/search/AsyncSearchErrorTraceIT.java

```diff
@@ -40,6 +43,12 @@ protected Collection<Class<? extends Plugin>> nodePlugins() {
     @Before
     public void setupMessageListener() {
         transportMessageHasStackTrace = ErrorTraceHelper.setupErrorTraceListener(internalCluster());
+        updateClusterSettings(Settings.builder().put(SearchService.BATCHED_QUERY_PHASE.getKey(), false));
```

Comment:
is this also a case that we'll want to fix as a follow-up? Would you mind adding a comment about why we need to turn batching off?

Reply:
Yea maybe on the fix topic, it's a test only issue fortunately, it's just that the assertion mechanism in here is very specific to unbatched network requests and won't work with the batched approach.
I'll add a comment :)